bukuserver/README.md

```diff
@@ -149,7 +149,7 @@ Note: the value for BUKUSERVER_REVERSE_PROXY_PATH
 is recommended to include preceding slash and not have trailing slash
 (i.e. use `/foo` not `/foo/`)
 
-### Screenshots
+### Screenshots ([imgur album](https://imgur.com/a/nk9dPEk))
```

Comment:
Don't add the link to the title; it changes title slug, which breaks links pointing at it.

Reply:
That is, I'm suggesting to have a separate paragraph after the section title, pointing to the Imgur album (preferably with somewhat more elaboration on the subject of the gallery).